translations/en-us.yaml

```diff
@@ -3343,16 +3343,51 @@ viewEditProject:
     Anybody with access to the API/UI acts as an admin and will be able to use any environment.
 
 volumesPage:
+  title: Volume
   noMatch: No volumes match the current search
+  banner:
+    driver: Driver
+    size: Size (Mb)
```

Comment:
`MB` (or possibly `MiB` if it's powers of 2, but probably not).  `Mb` is megabits.

Also would be nice to convert to format and convert to Gigs.  There isn't quite a function in util for that but `Util.formatSi(in_mb, 1000, "B", "B", 2)`